CalibTracker/SiStripChannelGain/plugins/SiStripGainFromCalibTree.cc

```diff
@@ -196,6 +193,7 @@ class SiStripGainFromCalibTree : public ConditionDBWriter<SiStripApvGain> {
         enum statistic_type {None=-1, StdBunch, StdBunch0T, FaABunch, FaABunch0T, IsoBunch, IsoBunch0T, Harvest};
 
         std::vector<string>    dqm_tag_;
+        std::vector booked_dir_;
```

Comment:
Turns out you can't reuse branch names in GitHub. After several tries I always get the code changes from the original and not the newest version.